CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+
+add_library(capsense INTERFACE)
```

Comment:
Why capsense is added as INTERFACE library, as opposed to other options defined in https://cmake.org/cmake/help/latest/command/add_library.html?